lib/process-services-cloud/src/lib/task/task-header/components/task-header-cloud.component.ts

```diff
@@ -0,0 +1,276 @@
+            new CardViewTextItemModel(
+                {
+                    label: 'ADF_CLOUD_TASK_HEADER.PROPERTIES.CATEGORY',
+                    value: this.taskDetails.category,
+                    key: 'category',
+                    default: this.translationService.instant('ADF_CLOUD_TASK_HEADER.PROPERTIES.CATEGORY_DEFAULT')
+                }
+            ),
+            new CardViewDateItemModel(
+                {
+                    label: 'ADF_CLOUD_TASK_HEADER.PROPERTIES.CREATED',
+                    value: this.taskDetails.createdDate,
+                    key: 'created'
+                }
+            ),
+            new CardViewMapItemModel(
+                {
+                    label: 'ADF_CLOUD_TASK_HEADER.PROPERTIES.PARENT_NAME',
+                    value: parentInfoMap,
+                    key: 'parentName',
+                    default: this.translationService.instant('ADF_CLOUD_TASK_HEADER.PROPERTIES.PARENT_NAME_DEFAULT'),
+                    clickable: true
+                }
+            ),
+            new CardViewTextItemModel(
+                {
+                    label: 'ADF_CLOUD_TASK_HEADER.PROPERTIES.PARENT_TASK_ID',
+                    value: this.taskDetails.parentTaskId,
+                    key: 'parentTaskId'
+                }
+            ),
+            new CardViewDateItemModel(
+                {
+                    label: 'ADF_CLOUD_TASK_HEADER.PROPERTIES.END_DATE',
+                    value: this.taskDetails.claimedDate,
+                    key: 'endDate'
+                }
+            ),
+            new CardViewTextItemModel(
+                {
+                    label: 'ADF_CLOUD_TASK_HEADER.PROPERTIES.ID',
+                    value: this.taskDetails.id,
+                    key: 'id'
+                }
+            ),
+            new CardViewTextItemModel(
+                {
+                    label: 'ADF_CLOUD_TASK_HEADER.PROPERTIES.DESCRIPTION',
+                    value: this.taskDetails.description,
+                    key: 'description',
+                    default: this.translationService.instant('ADF_CLOUD_TASK_HEADER.PROPERTIES.DESCRIPTION_DEFAULT'),
+                    multiline: true,
+                    editable: true
+                }
+            ),
+            new CardViewTextItemModel(
+                {
+                    label: 'ADF_CLOUD_TASK_HEADER.PROPERTIES.FORM_NAME',
+                    value: this.formName,
+                    key: 'formName',
+                    default: this.translationService.instant('ADF_CLOUD_TASK_HEADER.PROPERTIES.FORM_NAME_DEFAULT'),
+                    clickable: !!this.formName,
+                    icon: 'create'
+                }
+            )
+        ];
+    }
+
+    /**
+     * Refresh the card data
+     */
+    refreshData() {
+        if (this.taskDetails) {
+            const parentInfo = this.getParentInfo();
+            const defaultProperties = this.initDefaultProperties(parentInfo);
```

Comment:
I don't get why we do like that.
can we remove this line 
 `const parentInfo = this.getParentInfo();`

and do`
new CardViewMapItemModel(
                {
                    label: 'ADF_CLOUD_TASK_HEADER.PROPERTIES.PARENT_NAME',
                    value: this.getParentInfo(),
                    key: 'parentName',
                    default: this.translationService.instant('ADF_CLOUD_TASK_HEADER.PROPERTIES.PARENT_NAME_DEFAULT'),
                    clickable: true
                }
            ),
`